### ISSUE TITLE
hyperdex change domain again and its blocked cloudflare

### ISSUE THREAD

**Issue by adrianoaxl1** [RedSquirrel87/manga-downloader#840]
(https://hiperdex.com/)
[jvm_log.txt](https://github.com/user-attachments/files/16630900/jvm_log.txt)

the site now use the (.com) again and now its blocked (cloudflare)

**Comment by RedSquirrel87** [comment]
Yes, unfortunately, even restoring the correct domain doesn’t work, the problem is indeed Cloudflare. I tested it and at the moment it cannot be bypassed even with the web extension

**Comment by adrianoaxl1** [comment]
Thanks for responding I hope with time it will work again we have to be patience, have a amazing day Em 17/08/2024 10:01, Red Squirrel ***@***.***> escreveu:
Yes, unfortunately, even restoring the correct domain doesn’t work, the problem is indeed Cloudflare. I tested it and at the moment it cannot be bypassed even with the web extension

**Comment by RedSquirrel87** [comment]
I just released the v64.5 that should fix this issue. Please remember to use the Firefox version of the addon since in Chrome, with the new manifest v3, the extension can not get the cloudflare bypass cookie anymore
